stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -93,6 +93,15 @@ public fun Messages(
     var parentSize by remember { mutableStateOf(IntSize(0, 0)) }
     val density = LocalDensity.current
 
+    /** Marks the bottom most item as read every time it changes. **/
+    OnLastVisibleItemChanged(lazyListState) { messageIndex ->
```

Comment:
One possible consequence I can think of here is having an out of bounds error but brief torture tests did not manage to cause it.

If you can think of a scenario, please test it.